build.gradle

```diff
@@ -124,4 +124,10 @@ processResources {
         include 'mcmod.info'
         expand 'version': project.version, 'mcversion': minecraft_version
     }
-}
\ No newline at end of file
+}
+
+idea { 
```

Comment:
You needed that for idea i suspect. Does not fully belong to this PR and i think @JosephRymer have managed to set up  idea without this. Ask him and maybe this is not needed.